testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -442,6 +480,11 @@ fn find_microblock_privkey(
     return None;
 }
 
+// Returns true if relative difference between `a` and `b` is less than 0.01.
+fn is_near(a: f64, b: f64) -> bool {
+    (a - b).abs() < 0.01
```

Comment:
Do you want to use `error_epsilon` constant from fee_medians.rs in here?

Reply:
I'm not sure where else it would go. I think it's alright to leave it in this file.

Reply:
just copy-and-pasted for now